src/sd_jwt/utils/demo_utils.py

```diff
@@ -54,17 +54,23 @@ def get_jwk(jwk_kwargs: dict = {}, no_randomness: bool = False, random_seed: int
     """
     if no_randomness:
         random.seed(random_seed)
-        issuer_key = JWK.from_json(json.dumps(jwk_kwargs["issuer_key"]))
+        issuer_keys = [JWK.from_json(json.dumps(k)) for k in jwk_kwargs["issuer_keys"]]
         holder_key = JWK.from_json(json.dumps(jwk_kwargs["holder_key"]))
         logger.warning("Using fixed randomness for demo purposes")
     else:
         _kwargs = {"key_size": jwk_kwargs["key_size"], "kty": jwk_kwargs["kty"]}
-        issuer_key = JWK.generate(**_kwargs)
+        issuer_keys = [JWK.generate(**_kwargs)]
         holder_key = JWK.generate(**_kwargs)
 
-    issuer_public_key = JWK.from_json(issuer_key.export_public())
+    if len(issuer_keys) > 1:
+        issuer_public_keys = JWKSet()
+        for k in issuer_keys:
+            issuer_public_keys.add(JWK.from_json(k.export_public()))
+    else:
+        issuer_public_keys = JWK.from_json(issuer_keys[0].export_public())
```

Comment:
I would normalize it to array anyway, and therefore let the code flow ... go

```suggestion
    if instance(issuer_keys, dict):
        issuer_keys = [issuer_keys]
    issuer_public_keys = JWKSet()
    for k in issuer_keys:
            issuer_public_keys.add(JWK.from_json(k.export_public()))
```

Reply:
Let's refactor later!